src/Data/SongDetailsContainer.cpp

```diff
@@ -60,20 +60,27 @@ namespace SongDetailsCache {
         }
 
         for (const auto& [src, _] : DataGetter::dataSources) {
-            auto db = DataGetter::UpdateAndReadDatabase(src).get();
-            if (!db.has_value()) {
-                break;
-            }
+            DEBUG("Trying source {}", src);
```

Comment:
please remove this one and just log if a source failed (at the catch below, just make it print the src there)